Tests/Serilog.Exceptions.Test/Destructurers/ReflectionBasedDestructurerTest.cs

```diff
@@ -242,8 +242,8 @@ public void WhenObjectContainsCyclicReferences_ThenNoStackoverflowExceptionIsThr
             var myObject = (Dictionary<string, object>)result.GetResultDictionary()["MyObject"];
 
             Assert.Equal("bar", myObject["Foo"]);
-            Assert.Equal(myObject["$id"], ((Dictionary<string, object>)myObject["Reference"])["$ref"]);
-            Assert.Equal(myObject["$id"], ((Dictionary<string, object>)myObject["Reference2"])["$ref"]);
+            Assert.Equal(myObject["$id"], ( (Dictionary<string, object>)myObject["Reference"] )["$ref"]);

```

Comment:
I fixed the .editorconfig file that caused this. Feel free to fix it or I'll do so.